python/ray/actor.py

```diff
@@ -392,6 +393,20 @@ def export_actor_class(class_id, Class, actor_method_names,
         "actor_method_names": json.dumps(list(actor_method_names))
     }
 
+    if (len(actor_class_info["class"]) >
+            ray_constants.PICKLE_OBJECT_WARNING_SIZE):
+        warning_message = ("Warning: The actor {} has size {} when pickled. "
+                           "It will be stored in Redis, which could cause "
+                           "memory issues. This may mean that the actor "
+                           "definition uses a large array or other object."
+                           .format(actor_class_info["class_name"],
+                                   len(actor_class_info["class"])))
+        ray.utils.push_error_to_driver(
+            worker.redis_client,
+            "pickling_large_object",
```

Comment:
should this be a constant somewhere?

Reply:
Yes it should be. I partially addressed this. However, this should really be done in flatbuffers.